var/migrations/Version20170614232630.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Fix typo in parameter anneeCreation
+ */
+class Version20170614232630 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('Update `AdminParam` set name = "anneeCreation" where name = "anneCreation"');
```

Comment:
It would probably be good to follow a proper SQL style:

```sql
UPDATE AdminParam SET name = 'anneeCreation' WHERE name = 'anneCreation';
```

(MySQL keywords in uppercase)